traffic_portal/app/src/traffic_portal_properties.json

```diff
@@ -178,71 +178,71 @@
-            "desc": "Cache URL rule to apply to this delivery service.<br><br>See <a href=\"https://docs.trafficserver.apache.org/en/latest/admin-guide/plugins/cacheurl.en.html\" target=\"_blank\">ATS documentation on cacheurl...</a>"
+            "desc": "Allows you to manipulate the cache key of the incoming requests. Normally, the cache key is the origin domain. This can be changed so that multiple services can share a cache key, can also be used to preserve cached content if service origin is changed.<br><b>Note:<b> Only valid in ATS 6. X and earlier. Must be empty if using ATS 7. X and / or the <a href = \"https://docs.trafficserver.apache.org/en/latest/admin-guide/plugins/cachekey.en.html\" target=\"_blank\">cachekey plugin.</a><br><br>See <a href = \"https://docs.trafficserver.apache.org/en/6.2.x/admin-guide/plugins/cacheurl.en.html\" target=\"_blank\">ATS documentation on cacheurl </a>"
           },
           "ccrDnsTtl": {
             "title": "Delivery Service DNS TTL",
-            "desc": "The Time To Live on the DNS record for the Traffic Router A and AAAA records."
+            "desc": "(For DNS routed delivery services only) The Time To Live on the DNS record for the Traffic Router A and AAAA records. Default value is 30 seconds. Setting too high or too low will result in poor caching performance."
           },
           "cdnId": {
             "title": "CDN",
-            "desc": "The CDN in which the delivery service belongs to."
+            "desc": "The CDN in which the delivery service belongs."
           },
           "checkPath": {
             "title": "Check Path",
             "desc": "A path (ex: /crossdomain.xml) to verify the connection to the origin server with. This can be used by Check Extension scripts to do periodic health checks against the delivery service."
           },
           "deepCachingType": {
             "title": "Deep Caching",
-            "desc": "When to do deep caching for this delivery service.<br><br>See <a href=\"http://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_router.html#rl-deep-cache\" target=\"_blank\">Deep Caching</a> for more details..."
+            "desc": "Enables clients to be routed to the closest possible deep edge caches on a per Delivery Service basis.<br><br><a href=\"http://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_router.html#rl-deep-cache\" target=\"_blank\">See Deep Caching</a>"
           },
           "displayName": {
             "title": "Display Name",
-            "desc": "A human-readable name for the delivery service."
+            "desc": "Name of the service that appears in the Traffic portal. No character restrictions."
           },
           "dnsBypassCname": {
             "title": "DNS Bypass CNAME",
-            "desc": "Description required"
+            "desc": "(For DNS routed delivery services only) Domain name to overflow requests when the the max Bps or Max Tps for this delivery service are exceeded."
```

Comment:
again, the (For DNS routed delivery services only) is redundant. and a couple typos in here.